DigitalLearningSolutions.Web/Styles/learningMenu/tutorial.scss

```diff
@@ -27,3 +27,18 @@
   width: 100%;
   height: auto;
 }
+
+.grid-column-85 {

```

Comment:
Is this a class in the nhs library or our own class? 🙂

Reply:
This is our own class, copied from what was used in the self-assessment. Without it the links end up pressed right against the edges of the screen when a narrow viewport is used 🙂

Reply:
Ah fair enough 🙂